bootstrap-demo/kubernetes-api/kube-apply.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+kubectl apply -f kubernetes/akka-cluster.yml
```

Comment:
Is that script still useful? Just showing the `kubectl apply` command seems more explicit and should be understandable by the intended audience (if that's something you can say about `kubectl apply` at all--when you read around the web, `kubectl apply` seems like the "starting exe files sent to you by strangers via email" of the modern systems administrator... ;) ).

Reply:
Yep, will remove these and just add the command